### PR TITLE
chore: Display of Employee Name instead of Employee Code

### PR DESCRIPTION
**Pull request by Mayur-Bheda-epi**
Asana Link : https://app.asana.com/0/1200358785304462/1200351727616747

image
image

### CODE REVIEW

File: erpnext/hr/doctype/leave_application/leave_application.js [hunk 1]
```diff
@@ -85,6 +86,9 @@ frappe.ui.form.on("Leave Application", {
 				frm.set_value('employee', perm['Employee'].map(perm_doc => perm_doc.doc)[0]);
 			}
 		}
+		if((!frm.doc.__islocal) && frm.doc.employee) {
+			 $('[data-fieldname="employee_name"]')[0].hidden = true
+		}
```
Comment (hrwX) [inline erpnext/hr/doctype/leave_application/leave_application.js]:
```suggestion
```
- Any reason for hiding this ?

File: erpnext/hr/doctype/employee/employee.json [hunk 2]
```diff
@@ -602,30 +602,30 @@
...
-   "hidden": 1
+   "options": "\nA+\nA-\nB+\nB-\nAB+\nAB-\nO+\nO-"
   },
   {
    "description": "Here you can maintain family details like name and occupation of parent, spouse and children",
    "fieldname": "family_background",
    "fieldtype": "Small Text",
-   "label": "Family Background",
-   "hidden": 1
+   "hidden": 1,
```
Comment (hrwX) [inline erpnext/hr/doctype/employee/employee.json:620]: - Can we unhide all the fields ?

**Review by hrwX — CHANGES_REQUESTED** [review]

File: erpnext/hr/doctype/leave_application/leave_application.js (hunk 1 above)
Comment (Mayur-Bheda-epi) [reply erpnext/hr/doctype/leave_application/leave_application.js]: There is a field named
1. employee 
2. employee_name 
At the time of New Record employee code is shown in employee field and name in employee_name
but at the time of Edit or view in both the fields employee name is showing and it is duplicate so i was hidding the employee_name

File: erpnext/hr/doctype/employee/employee.json (hunk 2 above)
Comment (Mayur-Bheda-epi) [reply erpnext/hr/doctype/employee/employee.json:620]: I have just changed the settings in last line rest is default changes . 
Unhide all the fields was not in the DOD so not touched, if require then we can make changes

Comment (sahil28297) [reply erpnext/hr/doctype/employee/employee.json:620]: @hrwX let's not change the old configurations, it was hidden earlier as well.

**Review by sahil28297 — CHANGES_REQUESTED** [review]

File: erpnext/hr/doctype/leave_application/leave_application.js (hunk 1 above)
Comment (hrwX) [reply erpnext/hr/doctype/leave_application/leave_application.js]: - This is a hack.
- Do not delete the field. just hide it in doctype

**Review by hrwX — CHANGES_REQUESTED** [review]

**Review by hrwX — APPROVED** [review]

**Review by sahil28297 — APPROVED** [review]

**Pull request merged**